notes_app/README.md

```diff
@@ -54,13 +54,34 @@ npm install
 npm run dev
 ```
 
+## Deployment
+
+### Frontend (Netlify)
+
+**1. This REACT_APP_BACKEND_URL needs to be set up in the .env file in the Frontend -> Deployment**, before run `npm run build`.
```

Comment:
say that the REACT_APP_BACKEND_URL needs to point to Railway URL and say where to get the Railway URL from